lib/cli/app/template/index.tpl.js

```diff
@@ -1,21 +1,15 @@
 'use strict';
 
-const controllers = require('hof-controllers');
-
 module.exports = {
   name: '{{name}}',
   baseUrl: '/{{name}}',
   steps: {
-    '/': {
-      controller: controllers.start,
-      next: '/name'
-    },
     '/name': {
       fields: ['name'],
       next: '/confirm'
     },
     '/confirm': {
-      controller: require('hof-confirm-controller'),
+      behaviours: ['complete', require('hof-behaviour-summary-page')],
```

Comment:
do you think this is the best order? I put the 'complete' after the SummaryPage behaviour - no real reason apart from 'complete' felt like a more final step!